### ISSUE TITLE
Escaping selector strings

### ISSUE THREAD

**Issue by jamesjjk** [tizmagik/react-head#16]
@tizmagik 

Hey you should consider also escaping selector strings:

```
/* eslint no-useless-escape:0 */
const buildSelector = obj =>
  Object.keys(obj)
    .map(k => {
      const cnt = obj[k].replace(/["\\]/g, '\\$&');
      return `[${k}="${cnt}"]`;
    })
    .join('');

export default buildSelector;
```

**Comment by tizmagik** [comment]
Good call, yes. Would you be up for submitting a PR?

**Comment by jamesjjk** [comment]
I have provided a PR for both #14 and Escaping strings. Please note that the build directory is included

**Comment by tizmagik** [comment]
@jamesjjk I left the escaping out is the recent PR. I wanted to make sure I understand the use case for this first and have a test for it. Do you have an example of a real world use case you hit for this? Also happy to accept a separate PR if you’re up for it. 

Thanks!

**Comment by jamesjjk** [comment]
@tizmagik Use case is for example a title with a double quote mark.

**Comment by tizmagik** [comment]
This is now fixed with the [latest v3.0.0 release](https://github.com/tizmagik/react-head/releases/tag/v3.0.0). Please feel free to open a new issue if there's still any problems.
